python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -128,14 +128,16 @@ def transform_file(
         or will overwrite a table given in the `output_table` parameter.
     """
 
-    @transform(
+    def null_function():
+        return file_path, parameters
+
+    return TransformOperator(
         conn_id=conn_id,
         parameters=parameters,
+        handler=None,
         database=database,
         schema=schema,
+        sql=file_path,
+        python_callable=null_function,
         **kwargs,
-    )
-    def transform_file_func(file_path: str):
-        return file_path
-
-    return transform_file_func(file_path)
+    ).output
```

Comment:
Would it be possible to make an operator that just has the SQL stuff without all the decorator stuff? I feel like that gives us a lot of unnecessary complexity for the transform_file use-case.

Reply:
..but we are not using the `transform` decorator anymore. It uses the `TransformOperator` which is a normal Airflow operator. Could you elaborate on what you think is unnecessary, please?